### PR TITLE
New B and Lambda_b decays added for HF upgrade production

### PR DESCRIPTION
**Pull request by btrzecia**
Create a new case: kHFYellowReport, for HF upgrade production with new B0  and Lambda_b decay channels forced.

### CODE REVIEW

**Comment by dberzano** [comment]
+test

File: EVGEN/AliDecayer.h [hunk 1]
```diff
@@ -20,7 +20,8 @@ typedef enum
     kAll, kNoDecayHeavy, kHardMuons, kBJpsi,  kBJpsiUndecayed,
     kWToMuon,kWToCharm, kWToCharmToMuon, kZDiMuon, kZDiElectron, kNeutralPion, kAllMuonic,
     kChiToJpsiGammaToMuonMuon, kChiToJpsiGammaToElectronElectron, kNoDecayBeauty, kPsiPrimeJpsiDiElectron,
-  kElectronEM, kGammaEM, kDiElectronEM, kBeautyUpgrade,kHadronicDWithV0,kHadronicDWithout4BodiesWithV0,kAllEM,
+  kElectronEM, kGammaEM, kDiElectronEM, kBeautyUpgrade,kHFYellowReport,
```
Comment (rbertens) [inline EVGEN/AliDecayer.h]: Hi, trivial question maybe, but here kHFYellowReport is added in between of the list of elements of Decay_t . Are we sure that at other points in the code, Decay_t isn't compared to integer values ?

Comment (dberzano) [reply EVGEN/AliDecayer.h]: This is a very good point. It must be added at the end of the enum definition in any case. @btrzecia please update the PR as requested by @rbertens.

File: EVGEN/AliDecayer.h [hunk 2]
```diff
@@ -20,9 +20,9 @@ typedef enum
     kAll, kNoDecayHeavy, kHardMuons, kBJpsi,  kBJpsiUndecayed,
     kWToMuon,kWToCharm, kWToCharmToMuon, kZDiMuon, kZDiElectron, kNeutralPion, kAllMuonic,
     kChiToJpsiGammaToMuonMuon, kChiToJpsiGammaToElectronElectron, kNoDecayBeauty, kPsiPrimeJpsiDiElectron,
-  kElectronEM, kGammaEM, kDiElectronEM, kBeautyUpgrade,kHFYellowReport,
+  kElectronEM, kGammaEM, kDiElectronEM, kBeautyUpgrade,
```
Comment (dberzano) [inline EVGEN/AliDecayer.h]: Could you just fix the alignment here?

File: EVGEN/AliDecayer.h (hunk 1 above)
Comment (btrzecia) [reply EVGEN/AliDecayer.h]: Hi, indeed that was not a safe solution, thanks for pointing this out.

**Comment by rbertens** [comment]
+1 I realize that this PR was never merged. From my side it's OK, but it would need also approval of the 2nd list of approvers.

**Comment by chiarazampolli** [comment]
+1

**Pull request merged**